ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -712,7 +706,8 @@ - (void)respondToShoveGesture:(UIPanGestureRecognizer *)shoveRecognizer
 }
 #pragma mark Standard Initializer
 
-- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
+{
```

Comment:
Scandalous haha. But seriously do you prefer { on its own line? It's something the iOS community has moved away from but used to be the way to go.  I don't really care but we should standardize on it either way.